README.md

```diff
@@ -9,6 +9,11 @@ kaboom.js is a JavaScript library that helps you make games fast and fun!
 ```html
 <script src="https://kaboomjs.com/lib/0.4.0/kaboom.js"></script>
 <script type="module">
+	
```

Comment:
Nice, but I'd put the the usage text under README#Usage section, this example is supposed to be the minimal to copy paste and does not have to show all usages.